eslint.config.js

```diff
@@ -0,0 +1,45 @@
+import globals from 'globals';
+import eslint from '@eslint/js';
+import typescriptParser from '@typescript-eslint/parser';
+import mochaPlugin from 'eslint-plugin-mocha';
+import reactPlugin from 'eslint-plugin-react';
+
+const GLOBALS_BROWSER_FIX = Object.assign({}, globals.browser, {
+  AudioWorkletGlobalScope: globals.browser['AudioWorkletGlobalScope '],
+});
+
+delete GLOBALS_BROWSER_FIX['AudioWorkletGlobalScope '];
```

Comment:
Apparently following little bug has not been fixed and the `browser globals` is throwing an error without removing the extra trailing space: https://github.com/sindresorhus/globals/pull/184